src/wstool/multiproject_cli.py

```diff
@@ -840,6 +843,20 @@ def cmd_set(self, target_path, argv, config=None):
                 "Config path does not match %s %s " % (config.get_base_path(),
                                                        target_path))
 
+        if options.unmanaged:
```

Comment:
also, I would like an error message if --unmanaged is used with any other parameter, since those parameters would get ignored.